openedx_events/learning/signals.py

```diff
@@ -196,3 +197,58 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.exam.attempt.submitted.v1
+# .. event_name: EXAM_ATTEMPT_SUBMITTED
+# .. event_description: Emitted when an exam attempt is submitted by a learner.
+# .. event_data: ExamAttemptData
+EXAM_ATTEMPT_SUBMITTED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.exam.attempt.submitted.v1",
+    data={
+        "exam_attempt_data": ExamAttemptData,
```

Comment:
Nit. I think `exam_attempt_data` could be a little verbose and `data` is a little redundant. What do you think about just `exam_attempt`? That matches the other events more closely.

Reply:
Yeah that makes sense, changing now